jdatetime/__init__.py

```diff
@@ -867,6 +877,15 @@ def tzinfo(self):
     @staticmethod
     def strptime(date_string, format):
         """string, format -> new datetime parsed from a string (like time.strptime())"""
+
+        # assure are glyphs are standard digits
+        dt_string = date_string
+        if sys.version_info < (3, 0, 0):
```

Comment:
`sys.version_info` has been checked [before](https://github.com/slashmili/python-jalali/blob/5468676b9fa9bdd151718118845f6d816825dda8/jdatetime/__init__.py#L34). Maybe we can avoid the duplicate and use a module variable (say `PY3`) to hint Python version, and check that here instead.

Reply:
Actually python 2 will be deprecated in a few months, we can remove support back then. So I think it doesn't worth the effort.

Reply:
It's correct that Python 2 is deprecated, but all those Python applications out there will continue to work, it's just that their VM won't receive new updates, and that's just one of the VMs (sure, the most commonly used one) :wink:
I think decision to drop Python 2 support for this library needs a bit more care, and we'll come up with plan for that later.